tools/markdown-magic/src/transforms/packageImportInstructionsTransform.cjs

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const { PackageName } = require("@rushstack/node-core-library");
+
+const {
+	formattedGeneratedContentBody,
+	formattedSectionText,
+	getPackageMetadata,
+	resolveRelativePackageJsonPath,
+} = require("../utilities.cjs");
+
+/**
+ * Generates a simple Markdown heading and contents with information about how to import from the package's export options.
+ *
+ * @param {string} packageName - Name of the package (fully scoped).
+ * @param {boolean} includeHeading - Whether or not to include the heading in the generated contents.
+ * @param {boolean} hasAlphaExport - Whether or not the package has an `/alpha` export to be denoted.
+ * @param {boolean} hasBetaExport - Whether or not the package has an `/beta` export to be denoted.
+ * @param {boolean} hasLegacyExport - Whether or not the package has an `/legacy` export to be denoted.
+ */
+const generatePackageImportInstructionsSection = (
+	packageName,
+	includeHeading,
+	hasAlphaExport,
+	hasBetaExport,
+	hasLegacyExport,
+) => {
+	const lines = [
+		"This package leverages [package.json exports](https://devblogs.microsoft.com/typescript/announcing-typescript-4-5-beta/#packagejson-exports-imports-and-self-referencing) to separate its APIs by support level.",
+		"For more information on the related support guarantees, see [API Support Levels](https://fluidframework.com/docs/build/releases-and-apitags/#api-support-levels).",
+		"", // Blank line
+		"To access the `public` ([SemVer](https://semver.org/)) APIs, import directly from the root of the package as normal.",
```

Comment:
Small suggestion: Should we also provide an example import for public? I understand it's straightforward but since we do it for other, it would be consistent.

Reply:
Updated :)